include/wx/pdfdc29.h

```diff
@@ -231,6 +231,12 @@ class WXDLLIMPEXP_PDFDOC wxPdfDCImpl: public wxDCImpl
   wxPrintData    m_printData;
   wxPdfMapModeStyle m_mappingModeStyle;
 
+  bool          m_cachedColourSet{false};
+  wxPdfColour   m_cachedPdfColour;
+  unsigned char m_cachedColourR{};
```

Comment:
Wouldn't it be simpler to have a single `wxUint32 m_cachedRGB` member initialized using `wxColour::GetRGB()`? I think it would make the code shorter and more clear.

BTW, I'm not sure why is alpha ignored here, but this is a different issue, of course...